### PR TITLE
CVE and PSIRT fix for vpc-block 5.0

### PR DESCRIPTION
**Pull request by prankulmahajan**

### CODE REVIEW

File: config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json [hunk 1]
```diff
@@ -3,18 +3,20 @@
...
             "status": "supported",
             "latest-revision": {
-                "revision": "2",
-                "release-date": "17 November 2022",
-                "cves": "CVE-2022-1304,CVE-2016-3709",
-                "golang": "1.18.6",
-                "ubi": "8.7-923",
+                "revision": "3",
+                "release-date": "08 January 2023",
+                "cves": "",
```
Comment (ashimagarg27) [inline config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json]: Add cves fixed in new version of ubi image:
expected value:
`"cves": "CVE-2022-42898"`

File: config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json [hunk 2]
```diff
@@ -3,18 +3,20 @@
...
             },
             "previous-revision": {
-                "revision": "1",
-                "release-date": "07 November 2022",
-                "cves": "",
+                "revision": "2",
+                "release-date": "17 November 2022",
+                "cves": "CVE-2022-1304,CVE-2016-3709",
                 "golang": "1.18.6",
                 "ubi": "8.6-994",
```
Comment (ashimagarg27) [inline config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json]: update ubi image version in previous-release as
`"ubi": "8.7-923"`

Also, update the "new-features" list in previous-release to an empty list as
`"new-features": []`

**Review by ashimagarg27 — CHANGES_REQUESTED** [review]

File: config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json (hunk 1 above)
Comment (prankulmahajan) [reply config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json]: Updated

File: config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json (hunk 2 above)
Comment (prankulmahajan) [reply config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json]: Updated

**Review by anandkarra — APPROVED** [review]

File: config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json [hunk 3]
```diff
@@ -3,24 +3,24 @@
         "5.0": {
             "status": "supported",
             "latest-revision": {
+                "revision": "3",
+                "release-date": "08 January 2023",
```
Comment (mssachan) [inline config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json]: Change date to `09 January 2023`

File: config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json [hunk 4]
```diff
@@ -3,24 +3,24 @@
...
+                "cves": "CVE-2022-42898",
```
Comment (mssachan) [inline config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json]: `"cves": "CVE-2022-42898",` ----->
`"cves": "CVE-2022-42898,CVE-2022-41717,CVE-2022-41720",`

and 

`"fixes": [
                    "Fixes CVE-2022-42898,CVE-2022-41717,CVE-2022-41720 and PSIRT PVR0406812,PVR0410879,PVR0410611"
                 ]`
-----> `"fixes": []`

**Review by mssachan — CHANGES_REQUESTED** [review]
LGTM. With a minor comment

**Review by mssachan — APPROVED** [review]
LGTM

**Pull request merged**